### PR TITLE
Home page content b

### PR DESCRIPTION
**Pull request by MenarSelamet**
-Layout made
-Responsive
-Tests will be done after all the components of the home page are done
-Every branch is responsible of it's own component, please don't look at the merged componentes above or beneath, they're for testing purposes. This branch is for component B only which is the yellow section :D

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
...
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        poppins: ['Poppins'],
+      },
+    },
   },
   plugins: [],
 };
```
Comment (symys) [inline tailwind.config.js:12]: need importing for poppins font
@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
:)))

**Review by symys — CHANGES_REQUESTED** [review]
well done, I can't wait merge this ♥

**Review by SohailMahmoud — APPROVED** [review]
Ready to be merged, well done

**Review by symys — APPROVED** [review]
well done!

**Pull request merged**